docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -5,110 +5,117 @@ Diffraction Changes
-- On the fitting tab of the EngDiff UI the background can be inspected whether the background subtraction box is checked or not.
+- Sequential fitting in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` now uses the output of the last successful fit (as opposed to the previous fit) as the initial parameters for the next fit.
+- An empty :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` is no longer saved if the user saves a project having previously had the interface open at some point in that session.
+- The help button on the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` points to the correct page, having been broken in the last release.
+- Using the Clear button on the Workspace widget while using the :ref:`Fitting tab <ui engineering fitting>` no longer causes issues when you try to load runs back in.
+- On the :ref:`Fitting tab <ui engineering fitting>` of the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` the background can be inspected whether the background subtraction box is checked or not.
 
 Single Crystal Diffraction
 --------------------------
 New features
 ############
-- New algorithm :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for integrating four-circle data from HB3A in detector space.
 - New algorithm :ref:`ApplyInstrumentToPeaks <algm-ApplyInstrumentToPeaks>` to update the instrument of peaks within a PeaksWorkspace.
-- New plotting script that provides diagnostic plots of SCDCalibratePanels output.
-- New plotting script that provides diagnositc plots of SCDCalibratePanels2 on a per panel/bank basis.
-- Exposed :meth:`mantid.api.IPeak.getCol` and :meth:`mantid.api.IPeak.getRow` to python
-- Added two integration methods to :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for simple cuboid integration with and without fitted background.
 - New algorithm :ref:`ConvertPeaksWorkspace <algm-ConvertPeaksWorkspace>` for quick conversion between PeaksWorkspace and LeanElasticPeaksWorkspace.
-- New definition file for D19 ILL instrument added.
 - New algorithm :ref:`FindGlobalBMatrix <algm-FindGlobalBMatrix>` that refines common lattice parameters across peak workspaces from multiple runs with a different U matrix (which encodes the orientation) per run.
+- New algorithm :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for integrating four-circle data from HB3A in detector space.
+- New plotting script that provides diagnostic plots of :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` on a per panel/bank basis.
+- Exposed :meth:`mantid.api.IPeak.getCol` and :meth:`mantid.api.IPeak.getRow` to python.
+- Added two integration methods to :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for simple cuboid integration with and without fitted background.
```

Comment:
Maybe put the two notes relating to the `HB3AIntegrateDetectorPeaks` together - or even better see if they can be combined. e.g. 

New algorithm :ref:`HB3AIntegrateDetectorPeaks <algm-HB3AIntegrateDetectorPeaks>` for integrating four-circle data from HB3A in detector space using simple cuboid integration with and without fitted background.

This is not for ISIS instruments though so perhaps it's best to check the above with @zjmorgan ?

Reply:
That is correct. It is for HFIR only.

Reply:
I have combined the two notes in the first revision commit. Please can you check that it is correct @zjmorgan ?